src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```diff
@@ -1129,40 +1129,38 @@ private void callSpawnEvent(EntityMock entity, CreatureSpawnEvent.@NotNull Spawn
 			}
 
 
-			canceled = new CreatureSpawnEvent(living, reason).callEvent();
+			canceled = !new CreatureSpawnEvent(living, reason).callEvent();
 		}
 		else if (entity instanceof Item item)
 		{
-			canceled = new ItemSpawnEvent(item).callEvent();
+			canceled = !new ItemSpawnEvent(item).callEvent();
 		}
 		else if (entity instanceof Player)
 		{
 			canceled = true; // Shouldn't ever be called here but just for parody.
 		}
 		else if (entity instanceof Projectile)
 		{
-			canceled = new ProjectileLaunchEvent(entity).callEvent();
+			canceled = !new ProjectileLaunchEvent(entity).callEvent();
 		}
 		else
 		{
-			canceled = new EntitySpawnEvent(entity).callEvent();
+			canceled = !new EntitySpawnEvent(entity).callEvent();
 		}
 
-		/* EntityMock#getPassengers() and #getVehicle() isn't implemented
-		if (canceled || entity.isValid())
+		if (canceled || !entity.isValid())
```

Comment:
Also shouldn't this be "cancelled" with a double L?

Reply:
[Both are correct](https://www.grammarly.com/blog/canceled-vs-cancelled/). It just depends where you're from.

Reply:
Ah, not sure if I've ever noticed that before. Fixed.